api/filter_dimensions.go

```diff
@@ -222,6 +223,7 @@ func (api *API) updateFilterDimension(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// ID/name is provided by the request, but the label is provided by Cantabular.
+	// TODO: WHere is this used/necessary?
```

Comment:
Should we just remove it, if this is not used?

Reply:
I can't work out how, but the tests to break if I remove it